tests/slr_jest/admin/maps/actions.spec.js

```diff
@@ -0,0 +1,188 @@
+import {
+	getString,
+	registerDeleteAction
+} from '@tec/tickets/seating/admin/maps/actions';
+
+require('jest-fetch-mock').enableMocks();
+
+const locationBackup = window.location;
+
+function mockWindowLocation() {
+	delete window.location;
+	window.location = {
+		reload: jest.fn(),
+	};
+}
+
+function getTestDocument() {
+	return new DOMParser().parseFromString(
```

Comment:
Do not use a "syntetic" document in the tests, look at [the `getTestDocument` function](https://github.com/the-events-calendar/event-tickets/blob/1d0fd60e5de8741cb06a7762323cd2fd1038bdef/tests/slr_jest/_bootstrap.js#L170) and add the the snapshot (produced from PHP) you need to the list.
[I suggest this one](https://github.com/the-events-calendar/event-tickets/blob/92789c48eedfb10035d57209def09d6b1cd74ff8/tests/slr_integration/Admin/__snapshots__/Maps_Layout_Homepage_Test__test_maps_tab_card_listing_with_1_map__0.snapshot.html#L1-L0).

This is done to make sure changes to the PHP code that would break JS will be triggered by PHP snapshot updates.